src/pymedphys/sinogram/_level1/sinogram.py

```diff
@@ -39,22 +39,34 @@
 
```

Comment:
This file exists both in the labs and here? Where would you like it to be?

Reply:
@SimonBiggs Thought I had deleted the duplicate. The goal is to move it into the labs. Thanks.